source/state_representation/test/tests/test_cartesian_state.cpp

```diff
@@ -1,9 +1,9 @@
+#include <gtest/gtest.h>
+#include <cmath>
+
```

Comment:
should we have a linebreak here? It is not in every files but I am not sure what should be the convention

Reply:
@eeberhard ?

Reply:
The style guide convention is actually to break the include modules by scope (standard / system lib, then external headers, then internal / organisation headers that are not part of the actual build package, and finally "local" includes that _are_ part of the project build package). It looks a bit silly when there are not many includes, but it does help for readability of the dependencies I guess.

I wouldn't explicitly go and fix every other file to have the line breaks, but if we add them during other refactors it should be fine

Reply:
ok good to know I'll make sure to think about it

Reply:
So the way it is now is correct right?